package.json

```
@@ -3,7 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev -H local.climingo.vercel.app",
```

dev, stg, prd 호출할 script를 만들면 좋을 것 같아요!
```suggestion
    "dev": "next dev -H local.dev-climingo.vercel.app",
    "stg": "next dev -H local.stg-climingo.vercel.app",
    "prd": "next dev -H local.climingo.vercel.app",
```

---

"dev, stg, prd 호출할 script" 이게 어떤 의미일까요..? hostname이 변경되면 다른 api를 호출하나요?

---

현재 운영 서버를 dev, stg, prd client 서버 모두가 바라보고 있는데, 추후에 서버에서 dev, stg가 생성되면, 배포된 client도 바라보는 서버를 변경해야하고 로컬에서도 마찬가지로 어느 서버를 바라볼지 옵션으로 선택할 수 있게되면 좋을 것 같습니다!

---

서버가 제공되면 만들어도 될 것 같긴한데.. 우선 반영했어요! http에서 dev 서버 실행하는 스크립트는 제거했습니다! [1a45076](https://github.com/SingTheCode/climingo-client/pull/66/commits/1a45076c9ba5841c594ce7c86a9ce06d138d2c24)